packages/autorest.python/autorest/codegen/templates/model_dpg.py.jinja2

```diff
@@ -3,7 +3,7 @@
 
 
 {{ serializer.declare_model(model) }}
-    """{{ model.description(is_operation_file=False) | wordwrap(width=95, break_long_words=False, break_on_hyphens=False, wrapstring='\n    ')  }}
+    """{{ model.description(is_operation_file=False) | replace("\\", "\\\\") | wordwrap(width=95, break_long_words=False, break_on_hyphens=False, wrapstring='\n    ')  }}
```

Comment:
@onlined instead of copy-pasting this `replace` in multiple locations, can you update our macro for `wrap_string` (see code [here](https://github.com/Azure/autorest.python/blob/main/packages/autorest.python/autorest/codegen/templates/operation_tools.jinja2#L1)) to have this replace string, and call to `wrap_string` instead of `jinja2`s `wordwrap`? This way we get rid of code duplication.

To add the import so you can use `wrap_string`, make sure each template has this line at the top

```jinja2
{% import 'operation_tools.jinja2' as op_tools %}
```

Then when you're calling `wrap_string`, call it as `op_tools.wrap_string(...)`

Reply:
I didn't know if it would be OK to do that, as `wrap_string` might have called also for other purposes than docstring generating. Doing that right now, thanks for the comment!

Reply:
Done.

Reply:
@onlined can you make sure when you call `wrap_string` you prefix it with `op_tools.` and that you include the `operation_tools.jinja2` import at the top? thanks!